drift/app.py

```diff
@@ -1,13 +1,51 @@
-import connexion
+import os
 import logging
 
+import connexion
+import watchtower
+from boto3.session import Session
+
 from drift import config
 from drift.views import v1
 from drift.error import handle_http_error
 from drift.exceptions import HTTPError
 from drift.metrics_registry import create_prometheus_registry_dir
 
 
+def _setup_cw_logging(logger):  # pragma: no cover
```

Comment:
I know its a simple method but I'd benefit from this in a separate cloudwatch file of some sort.